src/scss/components/_scenarios.scss

```diff
@@ -1,7 +1,15 @@
+* {
```

Comment:
If you want to limit your component to a certain width, please use a class at the level of your component and affect the elements you want to affect. Otherwise you will affect all the elements in the page. Do not use the selector `*` unless it is something global across the site, and you are sure it's not going to break anything. Right now, because of this the site layout and design is breaking.

Reply:
Removed and replaced with the correct information in the mobile section and in desktop as well